projects/RabbitMQ.Client/client/impl/ModelBase.cs

```diff
@@ -608,7 +608,7 @@ public void HandleBasicCancelOk(string consumerTag)
                 _consumers.Remove(consumerTag);
             }
             ConsumerDispatcher.HandleBasicCancelOk(k.m_consumer, consumerTag);
-            k.HandleCommand(null); // release the continuation.
+            k.HandleCommand(IncomingCommand.Empty); // release the continuation.
```

Comment:
I don't want to imply anything needs to be changed. I'm asking for my own benefit.

Why `.Empty` instead of just `default`? My preference would have been `default` because I don't think `.Empty` conveys much especially given `.Empty` is actually `default`

Reply:
Honestly I didn't think too much about it. I do it for all "empty instances", but this one is actually a struct, so I'm uncertain about the consequences here. I'll need to investigate but my assumption is that default will (apart from the visibility) even be better overall.

Reply:
hmm interesting. My benchmark suggests that the Empty is faster here. Probably it is due to the fact of the `in` of the HandleCommand. The default case will have to create a temporary local to pass in and thus zeroing the stack for the "big" struct. Where as in the Empty case, the method just takes a reference of the static Empty field.